beacon-chain/operations/attestations/prepare_forkchoice.go

```diff
@@ -101,17 +102,40 @@ func (s *Service) seen(att *ethpb.Attestation) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	savedBits, ok := s.forkChoiceProcessedRoots.Get(string(attRoot[:]))
-	if ok {
-		savedBitlist, ok := savedBits.(bitfield.Bitlist)
+	savedBits, hasRoot := s.forkChoiceProcessedRoots.Get(string(attRoot[:]))
```

Comment:
Could savedBits here be nil and cause the next line to panic?

Reply:
Nope, verified it run time it doesn't panic, because of the type assertion
```
<nil>
<nil>
```